modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaFDSDataProviderCheck.testjava

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+import com.liferay.frontend.data.set.provider.FDSDataProvider;
+
+/**
+ * @author Michael Cavalcanti
+ */
+public class UpgradeJavaFDSDataProviderCheck implements FDSDataProvider {
+
+	@Override
+	public List getItems(FDSKeywords fdsKeywords, FDSPagination pagination, HttpServletRequest httpServletRequest, Sort sort)
+		throws PortalException {
+
+		return _original.getItems(fdsKeywords, pagination, httpServletRequest, sort
+		).stream(
+		).map(
+		).collect(
+			Collectors.toList()
+		);
```

Comment:
This is a strange usage/example. Technically, it is not valid Java code. Also, line 25 breaks the max 80 column rule. For this check, the 80 column rule is not irrelevant to the check, so we need to format it.